### PR TITLE
Forward-port of 4.0.6 changelog changes

### PR DESCRIPTION
**Pull request by astrofrog**
### Description

This are the changelog sections from the 4.0.6 release I just did

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [ ] Do the proposed changes actually accomplish desired goals?
- [ ] Do the proposed changes follow the [Astropy coding guidelines](https://docs.astropy.org/en/latest/development/codeguide.html)?
- [ ] Are tests added/updated as required? If so, do they follow the [Astropy testing guidelines](https://docs.astropy.org/en/latest/development/testguide.html)?
- [ ] Are docs added/updated as required? If so, do they follow the [Astropy documentation guidelines](https://docs.astropy.org/en/latest/development/docguide.html#astropy-documentation-rules-and-guidelines)?
- [ ] Is rebase and/or squash necessary? If so, please provide the author with appropriate instructions. Also see ["When to rebase and squash commits"](https://docs.astropy.org/en/latest/development/when_to_rebase.html).
- [ ] Did the CI pass? If no, are the failures related? If you need to run daily and weekly cron jobs as part of the PR, please apply the `Extra CI` label.
- [ ] Is a change log needed? If yes, did the change log check pass? If no, add the `no-changelog-entry-needed` label. If this is a manual backport, use the `skip-changelog-checks` label unless special changelog handling is necessary.
- [ ] Is a milestone set? Milestone must be set but `astropy-bot` check might be missing; do not let the green checkmark fool you.
- [ ] At the time of adding the milestone, if the milestone set requires a backport to release branch(es), apply the appropriate `backport-X.Y.x` label(s) *before* merge.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,4 +1,6 @@
 4.3.1 (2021-08-11)
+4.0.6 (2021-08-18)
+4.0.6 (2021-08-24)
```
Comment (pllim) [inline CHANGES.rst:3]: This looks weird.

**Review by pllim — COMMENTED** [review]
Also inconsistent spacing between entries in the diff.

**Comment by astrofrog** [comment]
Sorry cherry-pick failed massively. I had to just copy/paste the new section for 4.0.6... Should be fixed now.

**Review by pllim — APPROVED** [review]
LGTM. Thanks!

**Comment by bsipocz** [comment]
Didn't we say that the LTS changelog is not forward ported any more but keep totally separate, to avoid having duplicated entries?

Cc @Cadair

**Comment by Cadair** [comment]
That was also my recollection.

**Comment by astrofrog** [comment]
Ah sorry we can just close this then. However note there are two deleted fragments which maybe we should still delete from main then? Or maybe they will get deleted with 4.3.2. My brain hurts haha

**Comment by bsipocz** [comment]
deleting the fragments now that it's actively on our mind is I think better than wait with it until a new release.

**Comment by astrofrog** [comment]
I need to check but the fact the fragments are still here might indicate they were not included in 4.3.1 and are instead scheduled for 4.3.2. In this case we shouldn't remove the yet, right? (Otherwise they won't appear in any changelog section if we don't include LTS releases in the changelog)

**Comment by pllim** [comment]
So, should this PR be closed without merged? What is the path forward to sync the change logs?

**Comment by astrofrog** [comment]
I think it can be closed without merging but let's see what @eteq says

**Comment by eteq** [comment]
I agree closed without merging!  Since that seems to be consensus, I'm merging now.

**Pull request closed without merging**